libraries/js/immutable/list.effekt

```diff
@@ -20,6 +20,17 @@ def foreach[A](l: List[A]) { f: A => Unit } : Unit = {
   }
 }
 
+def foreachIndex[T](list: List[T]){ f: (Int, T) => Unit}: Unit = {
+  def worker[T](list: List[T], i: Int){ f: (Int, T) => Unit}: Unit = {
+    list match {
+      case Nil() => ();
+      case Cons(head, tail) => f(i, head); worker(tail, i+1){f}
```

Comment:
@IR0NSIGHT @phischu `f` is stable throughout the recursion, please consider dropping it from worker.